apollo.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     service: {
       name: 'MPDX',
       url: 'https://api.stage.mpdx.org/graphql',
+      notifyOnNetworkStatusChange: true,
```

Comment:
This isn't where it goes...

It's a field on query options. But instead of specifying it on every query you can default the query options with https://www.apollographql.com/docs/react/api/core/ApolloClient/#defaultoptions.

Add
```
    defaultOptions: {
      watchQuery: {
        notifyOnNetworkStatusChange: true,
      },
    },
```
after https://github.com/CruGlobal/mpdx-react/blob/main/src/lib/client.ts#L57 instead of here.

I don't see anything about including it in `apollo.config.js` https://www.apollographql.com/docs/devtools/apollo-config/#clientservice.